querybook/webapp/components/DataDocQueryCell/DataDocQueryCell.tsx

```diff
@@ -102,6 +102,7 @@ interface IState {
     query: string;
     meta: IDataQueryCellMeta;
 
+    modified: boolean;
```

Comment:
can you track it with a timestamp instead? so that we only trigger for past 5 minutes of action